app/src/main/java/swati4star/createpdf/fragment/SettingsFragment.java

```diff
@@ -0,0 +1,249 @@
+import static swati4star.createpdf.util.SettingsOptions.ImageEnhancementOptionsUtils.getEnhancementOptions;
+import static swati4star.createpdf.util.StringUtils.showSnackbar;
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+public class SettingsFragment extends Fragment implements OnItemClickListner {
+
+    @BindView(R.id.settings_list)
+    RecyclerView mEnhancementOptionsRecycleView;
+
+    private ArrayList<EnhancementOptionsEntity> mEnhancementOptionsEntityArrayList = new ArrayList<>();
+    private Activity mActivity;
+    private EnhancementOptionsAdapter mEnhancementOptionsAdapter;
+    private ImageToPDFOptions mPdfOptions;
+    private SharedPreferences mSharedPreferences;
+    private String mFontTitle;
+    private Font.FontFamily mFontFamily;
+    private int mFontSize;
+
+    public SettingsFragment() {
+        // Required empty public constructor
+    }
+
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+        mActivity = (Activity) context;
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        View root = inflater.inflate(R.layout.fragment_settings, container, false);
+        ButterKnife.bind(this, root);
+
+        mSharedPreferences = PreferenceManager.getDefaultSharedPreferences(mActivity);
+        mPdfOptions = new ImageToPDFOptions();
+        mFontTitle = String.format(getString(R.string.edit_font_size),
+                mSharedPreferences.getInt(Constants.DEFAULT_FONT_SIZE_TEXT, Constants.DEFAULT_FONT_SIZE));
+        mFontFamily = Font.FontFamily.valueOf(mSharedPreferences.getString(Constants.DEFAULT_FONT_FAMILY_TEXT,
+                Constants.DEFAULT_FONT_FAMILY));
+        mFontSize = mSharedPreferences.getInt(Constants.DEFAULT_FONT_SIZE_TEXT, Constants.DEFAULT_FONT_SIZE);
+
+        showEnhancementOptions();
+
+        return root;
+    }
+
+    private void showEnhancementOptions() {
+        GridLayoutManager mGridLayoutManager = new GridLayoutManager(getActivity(), 2);
+        mEnhancementOptionsRecycleView.setLayoutManager(mGridLayoutManager);
+        mEnhancementOptionsEntityArrayList = getEnhancementOptions(mActivity, mFontTitle, mFontFamily);
+        mEnhancementOptionsAdapter = new EnhancementOptionsAdapter(this, mEnhancementOptionsEntityArrayList);
+        mEnhancementOptionsRecycleView.setAdapter(mEnhancementOptionsAdapter);
+    }
+
+    @Override
+    public void onItemClick(int position) {
+        switch (position) {
+            case 0:
+                compressImage();
+                break;
+            case 1:
+                setPageSize();
+                break;
+            case 2:
+                editFontSize();
+                break;
+            case 3:
+                changeFontFamily();
+                break;
+        }
+    }
+
+    private void showCompression() {
```

Comment:
Please add comments to the newly added function this files in javadoc format.
For example.

```
/** 
 * Shows the dialog to change default compression value
 **/
```